01_fib/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fib(7)
+}
+
+func fib(n int) {
+
+	n1 := 0
+	n2 := 1
+
+	fibSeries := ""
+
+	for i := 0; i < n; i++ {
+		if n1 == 0 {
```

Comment:
Better to initialise `fibSeries` to avoid the need for this edge case.

Reply:
Done